### PR TITLE
Axisnow package

### PR DESCRIPTION
**Pull request by kristojorg**
This imports the axisnow decryptor package as a private npm package. The mock is still provided for when running without `NEXT_PUBLIC_AXISNOW_DECRYPT`, and we also fail at build time now if you try to run the app with decryption but don't have access to the module.

To get this to work, the axisnow package needs to be published to NPM and we need to update the `package.json` here to include it.

### CODE REVIEW

File: src/config/fetch-config.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ async function fetchConfigFile(configFileUrl) {
     const parsed = parseConfigText(text);
     return parsed;
   } catch (e) {
+    console.error(e);
```
Comment (kristojorg) [inline src/config/fetch-config.js:32]: Intended

File: src/utils/reader.ts [hunk 2]
```diff
@@ -82,18 +82,12 @@ async function initBookSettings(
...
   const scroller = new ScrollingBookView();
 
-  let decryptor: any = undefined;
-
-  try {
-    decryptor = Decryptor
-      ? await Decryptor.createDecryptor(decryptorParams)
-      : undefined;
-  } catch (e) {
+  if (!Decryptor) {
```
Comment (kristojorg) [inline src/utils/reader.ts:85]: If we don't have access but someone directly opens a reader page, they will now see this error.

**Comment by hokei** [comment]
we've wanted this for a while but it's gonna take me a bit to figure out where to get private npm publish permissions

**Comment by kristojorg** [comment]
Drats... I don't know how to make npm i ignore a dependency if we don't have access to it. Annoying

**Comment by hokei** [comment]
gah I guess we re-figured out why an npm module was tricky.  I'm sorry I couldn't remember it ...

**Comment by kristojorg** [comment]
Haha no worries but ohhhh well!

**Comment by kristojorg** [comment]
Gonna close this :(

**Pull request closed without merging**